spec/lib/open_project/text_formatting_spec.rb

```diff
@@ -61,28 +61,23 @@ def controller
 
       allow(User).to receive(:current).and_return(project_member)
       allow(Setting).to receive(:text_formatting).and_return('textile')
-
-      Setting.enabled_scm = Setting.enabled_scm << 'Filesystem' unless Setting.enabled_scm.include? 'Filesystem'
-    end
-
-    after do
-      allow(User).to receive(:current).and_call_original
-
-      Setting.enabled_scm.delete 'Filesystem'
     end
 
-    context 'Changeset links' do
-      let(:repository) { FactoryGirl.create :repository, project: project }
-      let(:changeset1) {
-        FactoryGirl.create :changeset,
-                           repository: repository,
-                           comments: 'My very first commit'
-      }
-      let(:changeset2) {
-        FactoryGirl.create :changeset,
-                           repository: repository,
-                           comments: 'This commit fixes #1, #2 and references #1 & #3'
-      }
+    context "Changeset links" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.